### ISSUE TITLE
LiveTV Metadata Not Fetched Natively

### ISSUE THREAD

**Issue by Artiume** [jellyfin/jellyfin#2336]
**Describe the bug**

My M3U/XLMTV Combination does not provide proper metadata for my shows so the details are blank except for the name. If I record a show, JF will properly fetch the metadata and will be available in my Recordings library, however, the metadata does not transfer to the livetv section.

**Comment by zehnerGIT** [comment]
Do you really have data in the recordings section (under Live-Tv)? How did you achieve that (because I tried to achieve that too)?
For me, this section seems to be a leftover (when GetRecordingsAsync() was part of ILiveTvService). Now recordings have to leverage IChannel and all recordings are only shown in their own library, but no longer in Live-Tv, so the recordings tab seems obsolete in Live-Tv.

**Comment by Artiume** [comment]
Yep, anything I record via livetv is popping up in the livetv recordings tab and inside the Recordings media section that's external to livetv. I hadn't even noticed the recordings tab inside livetv, I checked it and metadata is working there as well.

**Comment by Artiume** [comment]
Shoo bot

**Comment by Artiume** [comment]
Shoo bot

**Comment by srcrist** [comment]
Just to get some clarification, because this issue has sat for awhile: Shouldn't the metadata for the stream itself be embedded in the XML guide data? I'm not sure that I'm following this issue. Is anyone able to confirm that this issue still exists in 10.8.11 and provide some additional details?

**Comment by theguymadmax** [comment]
There is no plan to implement this. As this is not a bug, I'm closing this issue. You can create a feature request for it here:https://features.jellyfin.org/
